diff -Nur xsane-0.72.orig/Makefile.in xsane-0.72/Makefile.in
--- xsane-0.72.orig/Makefile.in	Thu Oct 19 23:34:47 2000
+++ xsane-0.72/Makefile.in	Sat Mar  3 02:26:29 2001
@@ -5,19 +5,19 @@
 top_srcdir = @top_srcdir@
 top_builddir = .
 
-prefix = @prefix@
-exec_prefix = @exec_prefix@
-bindir = @bindir@
-sbindir = @sbindir@
-libexecdir = @libexecdir@
-datadir = @datadir@
-sysconfdir = @sysconfdir@
-sharedstatedir = @sharedstatedir@
-localstatedir = @localstatedir@
-libdir = @libdir@
-infodir = @infodir@
-mandir = @mandir@
-includedir = @includedir@
+prefix = $(DESTDIR)@prefix@
+exec_prefix = $(DESTDIR)@exec_prefix@
+bindir = $(DESTDIR)@bindir@
+sbindir = $(DESTDIR)@sbindir@
+libexecdir = $(DESTDIR)@libexecdir@
+datadir = $(DESTDIR)@datadir@
+sysconfdir = $(DESTDIR)@sysconfdir@
+sharedstatedir = $(DESTDIR)@sharedstatedir@
+localstatedir = $(DESTDIR)@localstatedir@
+libdir = $(DESTDIR)@libdir@
+infodir = $(DESTDIR)@infodir@
+mandir = $(DESTDIR)@mandir@
+includedir = $(DESTDIR)@includedir@
 oldincludedir = /usr/include
 
 sanedatadir = @datadir@/sane
diff -Nur xsane-0.72.orig/doc/Makefile.in xsane-0.72/doc/Makefile.in
--- xsane-0.72.orig/doc/Makefile.in	Sat Dec 16 18:32:11 2000
+++ xsane-0.72/doc/Makefile.in	Sat Mar  3 02:27:22 2001
@@ -5,19 +5,19 @@
 top_srcdir = @top_srcdir@
 top_builddir = ..
 
-prefix = @prefix@
-exec_prefix = @exec_prefix@
-bindir = @bindir@
-sbindir = @sbindir@
-libexecdir = @libexecdir@
-datadir = @datadir@
-sysconfdir = @sysconfdir@
-sharedstatedir = @sharedstatedir@
-localstatedir = @localstatedir@
-libdir = @libdir@
-infodir = @infodir@
-mandir = @mandir@
-includedir = @includedir@
+prefix = $(DESTDIR)@prefix@
+exec_prefix = $(DESTDIR)@exec_prefix@
+bindir = $(DESTDIR)@bindir@
+sbindir = $(DESTDIR)@sbindir@
+libexecdir = $(DESTDIR)@libexecdir@
+datadir = $(DESTDIR)@datadir@
+sysconfdir = $(DESTDIR)@sysconfdir@
+sharedstatedir = $(DESTDIR)@sharedstatedir@
+localstatedir = $(DESTDIR)@localstatedir@
+libdir = $(DESTDIR)@libdir@
+infodir = $(DESTDIR)@infodir@
+mandir = $(DESTDIR)@mandir@
+includedir = $(DESTDIR)@includedir@
 oldincludedir = /usr/include
 configdir = ${sysconfdir}/sane.d
 sanedatadir = ${datadir}/sane
diff -Nur xsane-0.72.orig/intl/Makefile.in xsane-0.72/intl/Makefile.in
--- xsane-0.72.orig/intl/Makefile.in	Sun Sep 26 01:38:36 1999
+++ xsane-0.72/intl/Makefile.in	Sat Mar  3 02:29:24 2001
@@ -25,15 +25,15 @@
 top_builddir = ..
 VPATH = @srcdir@
 
-prefix = @prefix@
-exec_prefix = @exec_prefix@
+prefix = $(DESTDIR)@prefix@
+exec_prefix = $(DESTDIR)@exec_prefix@
 transform = @program_transform_name@
 libdir = $(exec_prefix)/lib
 includedir = $(prefix)/include
 datadir = $(prefix)/@DATADIRNAME@
 localedir = $(datadir)/locale
 gnulocaledir = $(prefix)/share/locale
-gettextsrcdir = @datadir@/gettext/intl
+gettextsrcdir = $(DESTDIR)@datadir@/gettext/intl
 aliaspath = $(localedir):.
 subdir = intl
 
diff -Nur xsane-0.72.orig/lib/Makefile.in xsane-0.72/lib/Makefile.in
--- xsane-0.72.orig/lib/Makefile.in	Tue Nov 21 23:10:50 2000
+++ xsane-0.72/lib/Makefile.in	Sat Mar  3 02:29:54 2001
@@ -5,19 +5,19 @@
 top_srcdir = @top_srcdir@
 top_builddir = ..
 
-prefix = @prefix@
-exec_prefix = @exec_prefix@
-bindir = @bindir@
-sbindir = @sbindir@
-libexecdir = @libexecdir@
-datadir = @datadir@
-sysconfdir = @sysconfdir@
-sharedstatedir = @sharedstatedir@
-localstatedir = @localstatedir@
-libdir = @libdir@
-infodir = @infodir@
-mandir = @mandir@
-includedir = @includedir@
+prefix = $(DESTDIR)@prefix@
+exec_prefix = $(DESTDIR)@exec_prefix@
+bindir = $(DESTDIR)@bindir@
+sbindir = $(DESTDIR)@sbindir@
+libexecdir = $(DESTDIR)@libexecdir@
+datadir = $(DESTDIR)@datadir@
+sysconfdir = $(DESTDIR)@sysconfdir@
+sharedstatedir = $(DESTDIR)@sharedstatedir@
+localstatedir = $(DESTDIR)@localstatedir@
+libdir = $(DESTDIR)@libdir@
+infodir = $(DESTDIR)@infodir@
+mandir = $(DESTDIR)@mandir@
+includedir = $(DESTDIR)@includedir@
 oldincludedir = /usr/include
 configdir = ${sysconfdir}/sane.d
 sanedatadir = ${datadir}/sane
diff -Nur xsane-0.72.orig/po/Makefile.in xsane-0.72/po/Makefile.in
--- xsane-0.72.orig/po/Makefile.in	Wed Mar  8 20:06:11 2000
+++ xsane-0.72/po/Makefile.in	Sat Mar  3 02:30:23 2001
@@ -17,8 +17,8 @@
 top_srcdir = @top_srcdir@
 VPATH = @srcdir@
 
-prefix = @prefix@
-exec_prefix = @exec_prefix@
+prefix = $(DESTDIR)@prefix@
+exec_prefix = $(DESTDIR)@exec_prefix@
 datadir = $(prefix)/@DATADIRNAME@
 localedir = $(datadir)/locale
 gnulocaledir = $(prefix)/share/locale
diff -Nur xsane-0.72.orig/src/Makefile.in xsane-0.72/src/Makefile.in
--- xsane-0.72.orig/src/Makefile.in	Thu Feb 15 19:39:28 2001
+++ xsane-0.72/src/Makefile.in	Sat Mar  3 02:31:05 2001
@@ -5,20 +5,20 @@
 top_srcdir = @top_srcdir@
 top_builddir = ..
 
-sane_prefix = @SANE_PREFIX@
-prefix = @prefix@
-exec_prefix = @exec_prefix@
-bindir = @bindir@
-sbindir = @sbindir@
-libexecdir = @libexecdir@
-datadir = @datadir@
-sysconfdir = @sysconfdir@
-sharedstatedir = @sharedstatedir@
-localstatedir = @localstatedir@
-libdir = @libdir@
-infodir = @infodir@
-mandir = @mandir@
-includedir = @includedir@
+sane_prefix = $(DESTDIR)@SANE_PREFIX@
+prefix = $(DESTDIR)@prefix@
+exec_prefix = $(DESTDIR)@exec_prefix@
+bindir = $(DESTDIR)@bindir@
+sbindir = $(DESTDIR)@sbindir@
+libexecdir = $(DESTDIR)@libexecdir@
+datadir = $(DESTDIR)@datadir@
+sysconfdir = $(DESTDIR)@sysconfdir@
+sharedstatedir = $(DESTDIR)@sharedstatedir@
+localstatedir = $(DESTDIR)@localstatedir@
+libdir = $(DESTDIR)@libdir@
+infodir = $(DESTDIR)@infodir@
+mandir = $(DESTDIR)@mandir@
+includedir = $(DESTDIR)@includedir@
 oldincludedir = /usr/include
 configdir = ${sysconfdir}/sane.d
 sanedatadir = ${datadir}/sane
