diff -ur xsane-0.991-o/doc/Makefile.in xsane-0.991/doc/Makefile.in
--- xsane-0.991-o/doc/Makefile.in	2006-01-01 06:08:11.000000000 -0700
+++ xsane-0.991/doc/Makefile.in	2006-07-17 15:52:17.000000000 -0600
@@ -21,7 +21,7 @@
 oldincludedir = /usr/include
 configdir = ${sysconfdir}/sane.d
 sanedatadir = ${datadir}/sane
-xsanedocdir = ${datadir}/sane/xsane/doc
+xsanedocdir = ${datadir}/xsane/doc
 
 MKINSTALLDIRS = @MKINSTALLDIRS@
 INSTALL = @INSTALL@
@@ -42,7 +42,7 @@
 
 %.1: %.man
 	sed -e 's|@DATADIR@|$(datadir)|g' \
-	    -e 's|@SANEDATADIR@|$(sanedatadir)|g' \
+	    -e 's|@SANEDATADIR@|$(datadir)|g' \
 	    -e 's|@CONFIGDIR@|$(configdir)|g' \
 	    -e 's|@LIBDIR@|$(libdir)|g' \
 	    -e 's|@BINDIR@|$(bindir)|g' \
diff -ur xsane-0.991-o/src/Makefile.in xsane-0.991/src/Makefile.in
--- xsane-0.991-o/src/Makefile.in	2006-01-06 16:22:47.000000000 -0700
+++ xsane-0.991/src/Makefile.in	2006-07-17 15:52:17.000000000 -0600
@@ -21,8 +21,8 @@
 includedir = @includedir@
 oldincludedir = /usr/include
 configdir = ${sysconfdir}/sane.d
-sanedatadir = ${datadir}/sane
-xsanedocdir = ${datadir}/sane/xsane/doc
+sanedatadir = ${datadir}
+xsanedocdir = ${datadir}/xsane/doc
 desktopappdir = @datadir@/applications
 pixmapdir = @datadir@/pixmaps
 
